go.mod

```diff
@@ -2,9 +2,11 @@ module github.com/nginxinc/nginx-service-mesh
 
 go 1.19
 
+replace github.com/servicemeshinterface/smi-controller-sdk => github.com/sjberman/smi-controller-sdk v0.0.0-20221228160702-f642bac3bdf9
```

Comment:
Why is this necessary?

Reply:
See the PR description.

Reply:
Sorry, haven't finished my coffee yet. 😄